app/assets/stylesheets/searchworks-mixins.css.scss

```diff
@@ -95,13 +95,18 @@
 }
 
 @mixin stackmap-find-it {
-  margin-left: 10px;
-
+  background-color: #007cbc;
+  border-color: #006ba3;
+  margin-right: 5px;
+  color: #ffffff;
```

Comment:
Can we move these colors into variables (or use existing ones when available)?  I believe that is the preferred practice in the scss world and will help us a lot when we implement SULStyles.

(FWIW `#ffffff` is already defined as `$white`)